### PR TITLE
Bot Data Socket Emit

### PR DESCRIPTION
**Pull request by tausifmuzaffar**
Create a Bot Data Socket emitter

### CODE REVIEW

**Comment by iamigo** [comment]
@tausifmuzaffar test failure?

File: db/model/botData.js [hunk 1]
```diff
@@ -111,6 +124,23 @@ module.exports = function botData(seq, dataTypes) {
...
+      afterCreate: (instance) => {
+        const changedKeys = Object.keys(instance._changed);
+        const ignoreAttributes = ['isDeleted'];
+        return realTime.publishObject(instance.toJSON(),
+          botDataEventNames.add, changedKeys, ignoreAttributes, pubOpts);
+      },
+
+      afterUpdate(instance /* , opts */) {
+        return realTime.publishObject(instance.toJSON(),
+          botDataEventNames.upd, null, null, pubOpts);
```
Comment (iamigo) [inline db/model/botData.js]: same question as the other pr about changed keys and ignore attributes in afterUpdate and afterDelete

Comment (tausifmuzaffar) [reply db/model/botData.js]: Fix this change

**Comment by tausifmuzaffar** [comment]
@pallavi2209 @pallavisingh9 this code is ready for review; also which username do you prefer?

**Comment by tausifmuzaffar** [comment]
@iamigo @kamilsmuga I think has some setting might have changed and only admins can merge PRs now?

**Pull request merged**